Gruntfile.js

```diff
@@ -240,7 +314,7 @@ module.exports = function (grunt) {
                 files: [{
                     expand: true,
                     src: ["src/**/css/stylus/*.styl"],
-                    ext: ".css",
+                    ext: "<% buildSettings.compress ? print('.min.css') : print('.css') %>",
```

Comment:
From my work before and reading the documentation at https://lodash.com/docs/4.17.4#template, I believe the `print` method is necessary when inside a lodash `evaluate` template delimiter to actually output a result.

Removing the print statements results in the `ext` parameter being blank (no file extensions); it seems necessary to use when we want to output based on evaluating JS in this style.